docs/getting_started.rst

```diff
@@ -0,0 +1,185 @@
+importing some files, to checking the contents of the database and exporting some files.
+
+1. Download and install Pepys
+-----------------------------
+
+Follow the instructions at :doc:`installation` to install Pepys.
+
+2. Copy configuration file
+--------------------------
+
+Copy the :code:`default_config.ini` file from the root of the Pepys installation, and place it in a
+location of your choice, and rename it to :code:`pepys_config.ini`. Open the file in a text editor
+and observe the configuration under the :code:`[database]` section: this configures which database
+to use. Edit the :code:`db_name` section to include a full path to a database file to create - you
+should ensure that this file does not already exist.
+
+3. Set environment variable
+---------------------------
+
+Create a new environment variable called :code:`PEPYS_CONFIG_FILE`, set to the full path to the
+:code:`pepys_config.ini` file that you copied in the previous step. Follow these `instructions
+<https://www.computerhope.com/issues/ch000549.htm>`_ to do this, but rather than editing the
+:code:`PATH` variable, choose to create a *New* variable, and configure the name and value
+appropriately.
+
+4. Run an import using Pepys Import
+-----------------------------------
+
+Navigate to the :code:`tests/sample_data/track_files/gpx` folder inside the Pepys install
+folder. Right click on :code:`gpx_1_0.gpx` and in the *Send To* menu choose :code:`Pepys Import
+(no archive)`. This will tell Pepys to import this GPX file into the database.
+
+A command-line window will open, showing the Pepys welcome banner, and then the 'resolver' will
+start asking you questions about bits of the data that it doesn't recognise. For example, it will
+ask you to create a new datafile, and then to create a new platform - as the platform in the GPX
+file wasn't in the database already. Use the number keys to choose an option each time you're asked
+a question, and type in relevant responses where necessary. There is no need to give real answers at
+the moment, this is just a first example.
+
+At the end of the import, a status table will be shown. It should look like this:
+
+.. code-block:: none
+
+    | Table name   |   Number of rows | Last item added            |
+    |--------------|------------------|----------------------------|
+    | States       |                5 | 2020-04-02 14:10:59.119746 |
+    | Contacts     |                0 | -                          |
+    | Comments     |                0 | -                          |
+    | Platforms    |                1 | 2020-04-02 14:10:54.507565 |
+
+*Note:* The dates will be different in your example, but the number of rows should be the same.
+
+5. Check the database status using Pepys Admin
+----------------------------------------------
+
+Run *Pepys Admin* from the Start Menu (you can either navigate to the *Pepys* folder and choose
+*Pepys Admin* or just search for it and press enter).
+
```

Comment:
@robintw - could you put in a reminder that if `Pepys Admin` isn't present, they should check they've run the correct steps from the installation guide.

Reply:
Fixed in d8fbe92